sql/core/src/main/scala/org/apache/spark/sql/sources/commands.scala

```diff
@@ -294,17 +294,16 @@ private[sql] abstract class BaseWriterContainer(
 
   private def newOutputCommitter(context: TaskAttemptContext): OutputCommitter = {
     val committerClass = context.getConfiguration.getClass(
-      "mapred.output.committer.class", null, classOf[OutputCommitter])
+      "mapreduce.output.committer.class", null, classOf[OutputCommitter])
```

Comment:
Since this is a Spark SQL only property, how about renaming it to `spark.sql.mapreduce.outputCommitterClass`? Current name looks like a genuine Hadoop property name, which is not true.